manifests/plugin/rpm.pp

```diff
@@ -0,0 +1,5 @@
+# @summary Pulp RPM plugin
+class pulpcore::plugin::rpm {
+  pulpcore::plugin { 'pulp_rpm':
```

Comment:
Looking at http://koji.katello.org/releases/yum/katello-nightly/pulpcore/el8/x86_64/python3-pulp-rpm-3.3.1-1.el8.noarch.rpm -- I think this should just be `rpm`